src/common/config_proxy.h

```diff
@@ -124,11 +124,16 @@ class ConfigProxy {
   ConfigValues* operator->() noexcept {
     return &values;
   }
-#ifdef WITH_SEASTAR
+  ConfigValues get_config_values() const {
```

Comment:
We already have static private `get_config_values` method, which is used only once in the copy constructor. It looks like it could be removed in favor of this one then?